### PR TITLE
Fix broken link of distributed in graphs.md

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix:
- As you can see in https://www.tensorflow.org/programmers_guide/graphs, the link of typical distributed configuration is broken:
    > If you are deploying TensorFlow in a @{$deploy/distributed$typical distributed configuration}, you might 

- The link of tensor-like type(https://www.tensorflow.org/programmers_guide/graphs#tensor_like_objects) is broken, changed it to https://www.tensorflow.org/programmers_guide/graphs#tensor-like_objects

### CODE REVIEW

File: tensorflow/docs_src/programmers_guide/graphs.md [hunk 1]
```diff
@@ -505,10 +505,10 @@ multiple graphs in the same process.
 As noted above, TensorFlow provides a "default graph" that is implicitly passed
 to all API functions in the same context. For many applications, a single graph
 is sufficient. However, TensorFlow also provides methods for manipulating
-the default graph, which can be useful in more advanced used cases. For example:
+the default graph, which can be useful in more advanced use cases. For example:
 
 * A @{tf.Graph} defines the namespace for @{tf.Operation} objects: each
-  operation in a single graph must have a unique name. TensorFlow will
+  operation in a single graph must have an unique name. TensorFlow will
```
Comment (frankchn) [inline tensorflow/docs_src/programmers_guide/graphs.md]: `a unique` is correct. The rule is to use `an` for words where the first syllable is a vowel sound. The first syllable of `unique` has a consonant sound, so `a` is correct.

**Review by frankchn — DISMISSED** [review]

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by imsheridan** [comment]
Is there any admin help review this patch?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by imsheridan** [comment]
@frankchn Is there any admin help review this patch?

**Review by martinwicke — APPROVED** [review]

**Pull request merged**